404.html

```diff
@@ -0,0 +1,159 @@
+  <meta property="og:type" content="website">
+  <meta property="og:image" content="exposure.png">
+
+  <meta name="twitter:site" content="@mimosaagency">
+  <meta name="twitter:title" content="Exposure">
+  <meta name="twitter:description" content="The simplest payment processing system.">
+  <meta name="twitter:url" content="https://exposure.money/">
+  <meta name="twitter:card" content="photo">
+  <meta name="twitter:image src" content="https://github.com/danilosierrac/exposure/blob/master/exposure.png?raw=true" />
+
+  <head profile="http://www.w3.org/2005/10/profile">
+  <link rel="icon" 
+      type="image/png" 
+      href="exposure.ico">
+
+<style>
+body {
+  font-family: -apple-system, "San Francisco", BlinkMacSystemFont, “Segoe UI”, Roboto, Helvetica, Arial, sans-serif, “Apple Color Emoji”, “Segoe UI Emoji”, “Segoe UI Symbol”; 
+  max-width: 80%; 
+  padding-top: 70px;
+  margin: auto;
+  font-size: 1.5em; 
+}
+
+input[type="text"] {
+  padding: 10px;
+  border: none;
+  text-align: center;
+  border-bottom: solid 2px #c9c9c9;
+  transition: border 0.3s
+}
+
+  input[type="text"]:focus,
+  input[type="text"].focus {
+  border-bottom: solid 2px #969696;
+  font-size: 1em;
+}
+
+  button 
+  {
+    background-color: #777;
+    border: none;
+    color: white;
+    padding: 12px 20px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 0px 2px;
+    cursor: pointer;
+  }
+
+hr 
+{
+   border: 0;
+    height: 2px;
+    background: #333;
+    background-color: #333;
+}
+
+h1{
+  font-size: 2em;
+}
+
+
+
+</style>
+</head>
+
+<body>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+
+  <div id="error" style="display:none;">
+    <h1>Whoops! Something went wrong with that link</h1>
+    <!-- TODO more explaination -->
+  </div>
+
+  <div id="pay" style="display: none;">
+    <h1>The simplest, minimal payment system that allows creatives to get paid in Exposure. 📈👍</h1>
+
+    <p>Just click the button below to pay <strong class="username">username</strong> the amount of <strong class="amount">0</strong> exposure!</p>
+
+    <button type="button" id="button">Pay with Exposure!</button>
+
+    <div style="margin-top: 30vh;">
+      <hr>
+      <p>Why the paypal link?</p>
+      <!-- TODO -->
+    </div>
+  </div>
+
+  <script type="text/javascript">
+    function showError() {
+      $("#error").show();
+      return false;
+    }
+
+    function showPaymentMessage(username, amount) {
+      $('#pay').find('.username').text(username);
+      $('#pay').find('.amount').text(amount);
+
+      $('#pay').show();
+    }
+
+    $(document).ready(function(){
```

Comment:
What lies below is the meat of this; It goes like:

1. get the pathname from the url; and split it by slashes
2. check that the second last part is "pay", this allows for either `/404.html/pay/:token` or `/pay/:token` to work
3. decode the token using [`atob`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob) — this gives a "random" looking url, but isn't actually.
4. Show the payment message which adds the two parts into the text.